twitter.go

```diff
@@ -51,10 +51,11 @@ func sendCurrentUsersTwitterMessage() {
 		for _, us := range MainRoom.users {
 			names = append(names, us.Name)
 		}
-		t, _, err := Client.Statuses.Update("People on Devzat rn: "+stripansi.Strip(fmt.Sprint(names))+"\nJoin em with \"ssh devzat.hackclub.com\"\nUptime: "+printPrettyDuration(time.Since(StartupTime)), nil)
+		tweetText := "People on Devzat rn: " + stripansi.Strip(fmt.Sprint(names)) + "\nJoin em with \"ssh devzat.hackclub.com\"\nUptime: " + printPrettyDuration(time.Since(StartupTime))
```

Comment:
```suggestion
		t, _, err := Client.Statuses.Update("People on Devzat rn: "+stripansi.Strip(fmt.Sprint(names))+"\nJoin em with \"ssh devzat.hackclub.com\"\nUptime: "+printPrettyDuration(time.Since(StartupTime)), nil)
```